lib/markdown_videos/renderer.rb

```diff
@@ -13,42 +13,26 @@ def initialize(string, options = {})
     end
 
     def render
-      MarkdownVideos::SERVICES.keys.each do |service|
-        @string = render_service(service)
-      end
-      @string
-    end
+      @string.gsub(/!\[([^\]]*)\]\(([^)]+)\)/) do
```

Comment:
`markdown_text` would maybe imply that it stays in `markdown` syntax. But after `render` is called, `markdown_text` would contain HTML